plugins/image-prioritizer/readme.txt

```diff
@@ -2,7 +2,7 @@
 
 Contributors: wordpressdotorg
 Tested up to: 6.6
-Stable tag:   0.1.2
+Stable tag:   0.2.0
```

Comment:
Since this has only a bug fix, shouldn't it be `0.1.3` instead?

Reply:
Yeah, good point. Fixed in 6c9e5fa

Reply:
Milestone renamed: https://github.com/WordPress/performance/milestone/60